File: TaskManagerAngularApp/src/app/models/employeeTask.ts
import { Employee } from "./employee";

export interface EmployeeTask
{
    id? : number;
    title?: string;
    duration?: number; 
    summary?: string;
    status?: string;
    employee?: Employee;
    // employee?: Employee["name"];
         
    preparingDelete?: boolean;
    isEditing?: boolean;
}

File: TaskManager/Models/EmployeeTask.cs
namespace TaskManager.Models

{
    public class EmployeeTask
    {
        public long Id { get; set; }
        public string? Title { get; set; }
        public int? Duration { get; set; } 
        public string? Summary { get; set; }
        public string? Status { get; set; }
        // public Employee? Employee { get; set; }
    }
}
